src/AWS.jl

```diff
@@ -44,6 +44,23 @@ using ..AWSExceptions: AWSException
 const user_agent = Ref("AWS.jl/1.0.0")
 const aws_config = Ref{AbstractAWSConfig}()
 
+function amazonify(mapping, kwargs)
+    new_kwargs = Dict{String,Any}()
+
+    for kwarg in kwargs
+        str = string(kwarg.first)
+        _key = if haskey(mapping, str)
+            mapping[str]
+        else
+            str
+        end
+
+        new_kwargs[string(_key)] = kwarg.second
+    end
```

Comment:
I think `.first` and `.second` is accessing the internal fields, so I'd do
```suggestion
    for (k, v) in kwargs
        str = string(k)
        _key = if haskey(mapping, str)
            mapping[str]
        else
            str
        end

        new_kwargs[string(_key)] = v
    end
```